src/Nethermind/Nethermind.Network/P2P/Subprotocols/Snap/Messages/StorageRangesMessageSerializer.cs

```diff
@@ -71,7 +82,7 @@ public StorageRangeMessage Deserialize(IByteBuffer byteBuffer)
             stream.ReadSequenceLength();
 
             message.RequestId = stream.DecodeLong();
-            message.Slots = stream.DecodeArray(s => s.DecodeArray(DecodeSlot));
```

Comment:
Why not, just make it a method?

Reply:
🤔

Reply:
RlpStream too low to refer to these types; implementing here would be working on internals of RlpStream